tests/core/test_toolset_manager.py

```
@@ -560,3 +562,158 @@ def test_custom_runbook_catalogs_none(tmp_path):
+        '        deep: "{{ env.TEST_RESOLVED_VALUE }}"\n'
+    )
+
+    manager = ToolsetManager(custom_toolsets=[yaml_file])
+    custom = manager.load_custom_toolsets(["builtin"])
+    assert len(custom) == 1
+    override = custom[0]
+
+    target = _builtin_like_toolset()
+    target.override_with(override)
+
+    assert target.config["secret"] == "RESOLVED_XX"
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to: "secret"

(S105)